docs/hooks/hook_civicrm_themes.md

```diff
@@ -0,0 +1,63 @@
+# hook_civicrm_import
+
+## Summary
+
+This Hook is called when building a list of available themes for use within CiviCRM.
+
+## Definition
+
+    hook_civicrm_themes( &$themes )
+
+## Parameters
+
+-  array  $themes - array of theme information 
+    - ext: string (required)
+      The full name of the extension which defines the theme.
+      Ex: "org.civicrm.themes.greenwich".
+    - title: string (required)
+      Visible title.
+    - help: string (optional)
+      Description of the theme's appearance.
+    - url_callback: mixed (optional)
+       A function ($themes, $themeKey, $cssExt, $cssFile) which returns the URL(s) for a CSS resource.
+       Returns either an array of URLs or PASSTHRU.
+       Ex: \Civi\Core\Themes\Resolvers::simple (default)
+       Ex: \Civi\Core\Themes\Resolvers::none
+    - prefix: string (optional)
+      A prefix within the extension folder to prepend to the file name.
+    - search_order: array (optional)
+      A list of themes to search.
+      Generally, the last theme should be "*fallback*" (Civi\Core\Themes::FALLBACK).
+    - excludes: array (optional)
+      A list of files (eg "civicrm:css/bootstrap.css" or "$ext:$file") which should never
+      be returned (they are excluded from display).     - object being imported (for now Contact
+
+## Returns
+
+-   null
+
+## Availability
+
+-   This hook was first available in CiviCRM 5.16
+
+## Example
+
+```php
+     /*
+     * A theme is a set of CSS files which are loaded on CiviCRM pages.
+     */
+
+    function civitest_civicrm_themes( &$themes ) {
+      $themes[] = [
+        'name' => 'civielection',
+        'title' => 'civielection theme',
+        'prefix' => NULL,
+        'url_callback' => '\\Civi\\Core\\Themes\\Resolvers::simple',
+        'search_order' => [
+          0 => 'civielection',
+          1 => '_fallback_',
+        ],
+        'excludes' => [],
```

Comment:
A couple reactions:

1. The example looks right/valid.
2. The example feel a bit intimidating because it includes all the fields. I'd suggest doing two examples - first, a minimal one; then, a full/complex one.
3. Instead of the string `_fallback_`, use the constant `Civi\Core\Themes::FALLBACK`

Reply:
ok @totten updated

Reply:
Thanks, looks good. 👍